woocommerce/woocommerce-free-shipping-progress-bar-block
# Allow progress bar block to be added within Cart i2 block

Fixes #25

**Note**

In the first iteration, a merchant should be able to use the shipping progress bar block within the Cart i2 block. This PR aims to enable merchants to do so.

**Steps to test**

1. Check out this PR.
2. Create a test page.
3. Add the Cart i2 block.
4. Add the Free Shipping Progress Bar block to the Cart Totals section.
5. Ensure that the Free Shipping Progress Bar block is visible in the editor.
6. Save the page and look up the frontend.
7. Ensure that the Free Shipping Progress Bar block is visible in the frontend.
8. Adjust the product quantity.
9. Ensure that the progress bar changes based on the cart totals.

**Screenshot**

<table>
<tr>
<td valign="top" width="50%">Editor:
<br><br>

![27-editor](https://user-images.githubusercontent.com/3323310/138214214-0fa5f1f7-7765-4d06-98f2-879cc68405de.png)
</td>
<td valign="top" width="50%">Frontend:
<br><br>

![27-frontend](https://user-images.githubusercontent.com/3323310/138214228-f122071e-169c-47cf-8e52-e041af389eb5.png)
</td>
</tr>
</table>
